### PR TITLE
point to `@webxdc/types` and use `@webxdc/vite-plugins` for emulator

### LINKED ISSUE #51: Is this a sample project or a development tool?
The first couple of lines of README say

https://github.com/webxdc/hello/blob/a42c553ba14045d69e718e11d123d8f3c4a78a2d/README.md?plain=1#L3

But for a sample project this looks a little too convoluted, e.g. the ["Limitations" section in README](https://github.com/webxdc/hello/blob/a42c553ba14045d69e718e11d123d8f3c4a78a2d/README.md?plain=1#L24-L74) looks unnecessary for a sample project, this sounds more like it applies to the `webxdc.js` stub dev tool. This makes it look like it's required reading to know how this sample project works, which it isn't.
I think sample project must be as short and easy to swallow as possible. We have this in https://docs.webxdc.org/get_started.html#a-simple-example, but I don't know, it looks like we're saying that this is the repo of that simple example.

If it's a dev tool, we shouldn't be regarding it as a sample project in https://docs.webxdc.org/.

I think we need to clearly separate the stub `webxdc.js` and the sample `index.html` into different projects (also see "[Stop the copy-paste of `create-xdc.sh`, `webxdc.js`](https://support.delta.chat/t/stop-the-copy-paste-of-create-xdc-sh-webxdc-js-eruda-js/2448?u=wofwca)").

### PR DESCRIPTION
**Pull request by adbenitez**
close #51

### CODE REVIEW

**Review by Simon-Laux — APPROVED** [review]
Looks good to me, but let's wait for @r10s' approval

File: index.html
Comment (r10s) [inline index.html:8]: so, that results to one 404 in all cases?

**Review by r10s — COMMENTED** [review]
first of all, thanks a lot for the cleanup of the other repo!

for `webxdc.js` on some CDN:

i understand, why we do not want to copy the file, wrt maintenance etc.

however, by pulling a file from some CDN, the magic of simplicity and understanding things when not deep into js is also gone somehow, at least for ppl as me.

when following the instructions, you will also have `https://cdn.jsdelivr.net/gh/webxdc/vite-plugins@latest/src/webxdc.js` forever in your source code ...

and these instructions are also not what we're saying at https://webxdc.org/docs/#a-simple-example

and having two webxdc.js in the code is also a bit weird.

so, regarding the fact that webxdc.js is not changed that often, maybe it is better to stay with a copy.

File: index.html
Comment (adbenitez) [reply index.html:8]: one of them will always succeed, if you are in DC the real thing will load while in the browser the emulator is used

**Comment by adbenitez** [comment]
> when following the instructions, you will also have `https://cdn.jsdelivr.net/gh/webxdc/vite-plugins@latest/src/webxdc.js` forever in your source code ...

that is not a problem since it just silently fails to load inside DC where the app doesn't have internet




> however, by pulling a file from some CDN, the magic of simplicity and understanding things when not deep into js is also gone somehow, at least for ppl as me.

at the same time now it is easier for people that don't have experience with git, they can just download the index.html file and just with that file alone or copying the content of the example index.html can get it to work

**Comment by link2xt** [comment]
I have made follow-up PRs https://github.com/webxdc/hello/pull/63 (removing types and pointing to the source where they can be downloaded if needed) and https://github.com/webxdc/hello/pull/64 (updating webxdc.js once so it points to the correct source where webxdc.js is maintained).

**Comment by link2xt** [comment]
There is also an unrelated `release-webxdc.yml` in this PR, could be a separate PR.

**Pull request closed without merging**
